resources/assets/js/components/Fields/HasMany.vue

```diff
@@ -137,10 +139,11 @@
                     .then(function (result) {
```

Comment:
```suggestion
                    .then(result => {
```

Use arrow functions. That way, you won't need to specifically define the `$vm` variable, and you will be able to use `this`